modules/caddyhttp/requestbody/caddyfile.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2015 Matthew Holt and The Caddy Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package requestbody
+
+import (
+	"github.com/caddyserver/caddy/v2/caddyconfig/httpcaddyfile"
+	"github.com/caddyserver/caddy/v2/modules/caddyhttp"
+	"strconv"
+)
+
+func init() {
+	httpcaddyfile.RegisterHandlerDirective("request_body", parseCaddyfile)
+}
+
+func parseCaddyfile(h httpcaddyfile.Helper) (caddyhttp.MiddlewareHandler, error) {
+	rb := new(RequestBody)
+
+	for h.Next() {
+		// if not, they should be in a block
```

Comment:
This comment doesn't really make sense because the only thing being parsed is `max_size`.

You should reject unknown fields (you can use a `switch` statement with a `default` that returns an error). You should also reject any additional args on the same line (if we decide to only configure this via the block option, see below)

I'm not convinced that the only way to configure this should be via a block, because it only has one option. For example, the directive might be `max_request_body_size <size>` instead.

But on the other hand, aligning it with the JSON config isn't a bad idea either. It's just that we might never have any other options than just `max_size`, so it feels kinda strange to require a block.

I'm of two minds about it.

Anyways, the actual syntax as written would be:

```
request_body [<matcher>] {
	max_size <size>
}
```

Reply:
yes, exactly. Should I change to `max_request_body_size` or do we keep it this way?

Reply:
I'm not sure. I'll let @mholt chime in. I'm leaning towards keeping it as-is but I wanted to bring up the point of discussion.

Reply:
I think this is a less-commonly-used feature, so we're probably fine with the more verbose syntax for now. It also lets us add more options like replacements (etc.) in the future.